tests/test_routes.py

```diff
@@ -1,22 +1,84 @@
-# Standard library
 import unittest
 
-# Local
-from webapp.app import app
+from flask_testing import TestCase
 
+from webapp.app import db, create_app
+from webapp.models import CVE, Notice, Status
 
-class TestRoutes(unittest.TestCase):
-    def setUp(self):
-        app.testing = True
-        self.client = app.test_client()
 
-        return super().setUp()
+class TestRoutes(TestCase):
+    def create_app(self):
```

Comment:
Maybe name this something different from `create_app`, which is already used?

I can't actually see where you're using this method. Are you using it? Couldn't you just create the application as part of `setUp`?

Reply:
Done.